platform-sdk/swirlds-platform-core/src/testFixtures/java/com/swirlds/platform/test/fixtures/consensus/framework/validation/StateVerifiableValidation.java

```diff
@@ -6,8 +6,17 @@
 import com.swirlds.platform.state.signed.ReservedSignedState;
 import edu.umd.cs.findbugs.annotations.NonNull;
 
+/**
+ * Validate state has enough signatures so that it can be trusted to be valid
+ *
+ */
 public class StateVerifiableValidation implements ConsensusStateValidation {
 
+    /**
+     * Validate that the state has enough signatures.
+     *
+     * @param reservedSignedState the state to validate
+     */
     @Override
     public void validate(@NonNull ReservedSignedState reservedSignedState) {
         assertTrue(reservedSignedState.get().isVerifiable());
```

Comment:
assert statements in Turtle were replaced with org.assertj.core.api.Assertions. Replace all assert statements